scripts/test/test_arguments_with_equals.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
The following 3 files  look very similar, maybe it would make sense to extract common parts.

Reply:
Do you mean this? I am uncertain, if this is so easy with bash. Extracting stuff with bash is bit harder.
```
if [[ -n "$TEST_DEBUG_OPTIONS" ]]; then
  # We want to split on spaces here
  # shellcheck disable=SC2086
  set $TEST_DEBUG_OPTIONS
fi

set -euo pipefail

SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"/
PROJECT_ROOT_DIR="$SCRIPT_DIR/../.."
STARTER_SCRIPT_DIR="$PROJECT_ROOT_DIR/exasol_script_languages_container_tool/starter_scripts"

```